test/support/time.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Urito.Time do
+  def cast!(time) do
```

Comment:
If you use Ecto 2.1 I think you can avoid a lot of date/time pains since they support native Elixir date types, https://github.com/elixir-ecto/ecto/blob/v2.1/CHANGELOG.md#highlights